lib/cluster.js

```diff
@@ -27,7 +27,19 @@ function Worker(options) {
   if (options === null || typeof options !== 'object')
     options = {};
 
-  this.suicide = undefined;
+  this.exitedVoluntarily = undefined;
+
+  Object.defineProperty(this, 'suicide', {
+    get: function() {
+      // TODO: Print deprecation message.
+      return this.exitedVoluntarily;
+    },
+    set: function(val) {
+      // TODO: Print deprecation message.
+      this.exitedVoluntarily = val;
+    }
+  });
```

Comment:
For reference, people sometimes do things that are incompatible with getters (unfortunately), so for an already released stable branch, this guarantees it's not breaking.

See https://github.com/nodejs/node/issues/1591